conf.pri.mingw

```diff
@@ -0,0 +1,43 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_0_16/include)
```

Comment:
RC_0_16? Are you sure?

Reply:
I just moved this (yes, yes, outdated) code...
Of course, I can update it incidentally if you want.